poonam/JS/Assignment6_js/que1.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que1</title>
+    </head>
+    <body>
+        <h1>Name List</h1>
+        <script>
+        var input=[];
+        for(let i=0; i<=4; i++){
+            input[i]=prompt("Enter name:");
+        }
+        document.writeln("List of names: "+input);
+        document.writeln("<br>Name List:");
+        for(let i=0; i<=4; i++){
+            document.writeln("<br>Name: "+input[i]);
+        }
+    </script>
+    </body>
+</html>
```

Comment:
Add new end line in all files.

Reply:
which line mam?

Reply:
like in all files after closing of </html> add new line .